src/server/websocket.coffee

```diff
@@ -1,9 +1,10 @@
 # This implements the WebSocket network API for ShareJS.
 EventEmitter = require('events').EventEmitter
 WebSocketServer = require('ws').Server
-
 sessionHandler = require('./session').handler
 
+CLIENT_TIMEOUT = 30000
```

Comment:
Two things:

#### First

Would it be interesting to reference the `HEARTBEAT_INTERVAL` constant when trying to reconnect the WebSocket connection?

https://github.com/conversation/ShareJS/blob/557ee00d008d65a915bae3df2ba68a6dfea5f2e1/src/client/reconnecting_websocket.coffee#L52

Just to make sure one doesn't change the `CLIENT_TIMEOUT` to a value that's less than 30 seconds, otherwise, timeout would beat the heartbeat check, mistakenly dropping healthy connections.

#### Second

Adding a `# 30 seconds` on its right 👍 

```suggestion
CLIENT_TIMEOUT = 30000 # 30 seconds
```